bigchaindb/common/utils.py

```diff
@@ -97,6 +97,30 @@ def validate_all_keys(obj_name, obj, validation_fun):
     return
 
 
+def validate_all_value_for_key(obj, key, validation_fun):
+    """Validates value for all (nested) occurences of `key` in `obj`
+       using `validation_fun`
+
+        Args:
+            obj (dict): dictonary object.
+            key (str): key whose value is to be validated.
+            validation_fun (function): function used to validate the value
+            of `key`.
+
+        Returns:
+            None: indicates validation successfull
+
+        Raises:
+            ValidationError: `validation_fun` will raise this error on failure
+    """
+    for vkey, value in obj.items():
+        if vkey == key:
+            validation_fun(value)
+        elif type(value) is dict:
+            validate_all_value_for_key(value, key, validation_fun)
+    return
```

Comment:
If you agree with the previous comment, then you can omit the `return` here.

Reply:
@vrde I think I will omit `return` as it makes the code a little more homogenous to rest of the code base.